generators/app/index.js

```diff
@@ -13,20 +14,22 @@ module.exports = class extends Generator {
         type: 'input',
         name: 'appName',
         message:
-          "What's the name of your application? This will be the default page title."
+          "What's the name of your application? This will be the default page title.",
```

Comment:
Can we include parenthetical examples at the end of each question, to provide context for users of the generator who aren't already familiar with how these titles are used/our naming conventions?
For example, here are some current form titles and subTitles:
```
  title: 'Apply to add a dependent to your VA benefits',
  subTitle: 'VA Form 21-686c',
...
  title: 'Update your Education Benefits',
  subTitle: 'Form 22-5495',
```

Reply:
I added the `default`s to serve as the parenthetical examples, but `appName` is one I was looking for feedback on and forgot to ask. So thanks for speaking up. I'll take another stab at showing more verbose examples for this prompt in particular.